steps/keras/callbacks.py

```diff
@@ -34,6 +34,23 @@ def on_epoch_end(self, epoch, logs={}):
             K.set_value(self.model.optimizer.lr, self.gamma * K.get_value(self.model.optimizer.lr))
 
 
+class UnfreezeLayers(Callback):
+    def __init__(self, unfreeze_on_epoch, from_layer=0, to_layer=1):
+        self.unfreeze_on_epoch = unfreeze_on_epoch
+        self.from_layer = from_layer
+        self.to_layer = to_layer
+
+        self.epoch_id = 0
+        self.batch_id = 0
+
+    def on_epoch_end(self, epoch, logs={}):
+        if self.epoch_id == self.unfreeze_on_epoch:
+            for i, layer in enumerate(self.model.layers):
+                if i >= self.from_layer and i <= self.to_layer:
```

Comment:
@jakubczakon in this implementation I unfreeze two layers. Is that what we want? I thought that we wanted to unfreeze the embedding layer only.